EssentialFeed/EssentialFeed/Image Comment/Image Comment Feature/CommentAuthor.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  CommentAuthor.swift
+//  EssentialFeed
+//
+//  Created by Alok Subedi on 02/02/2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+public struct CommentAuthor: Equatable{
```

Comment:
Missing space before `{`

```suggestion
public struct CommentAuthor: Equatable {
```